blivet/util.py

```diff
@@ -116,12 +118,39 @@ def mount(device, mountpoint, fstype, options=None):
 
     return rc
 
-def umount(mountpoint):
-    try:
-        rc = run_program(["umount", mountpoint])
-    except OSError:
-        raise
+def umount(mountpoint, lazy=False, maxretry=3, retrysleep=1.0):
+    """ Unmount the given mountpoint.
 
+        :param str mountpoint: Path to unmount
+        :param bool lazy: When True do a lazy (-l) umount
+        :param int maxretry: Number of times to try the umount
```

Comment:
According to the logic of the code below, isn't this more ``maxtry`` than ``maxretry``?